tools/travis-before-install.sh

```diff
@@ -25,6 +25,13 @@ if [ -n "$INSTALL_PICKLE5" ]; then
   pip install pickle5
 fi
 
+if [ -n "$PPC64_LE" ]; then
+  wget -O openblas-power8.tar.gz https://www.dropbox.com/s/zcwhk7c2zptwy0s/openblas-v0.3.5-ppc64le-power8.tar.gz?dl=0
```

Comment:
I'm not too worried about getting something from Dropbox, but could you add some comments on how this OpenBLAS was built and who has access to that Dropbox folder? Or a link to those details?

Reply:
I added a comment block